kornia/augmentation/container/augment.py

```diff
@@ -241,15 +271,23 @@ def identity_matrix(self, input: Tensor) -> Tensor:
 
     def inverse(  # type: ignore[override]
         self,
-        *args: DataType,
+        *args: Union[DataType, Dict[str, DataType]],
         params: Optional[List[ParamItem]] = None,
         data_keys: Optional[Union[List[str], List[int], List[DataKey]]] = None,
-    ) -> Union[DataType, List[DataType]]:
+    ) -> Union[DataType, List[DataType], Dict[str, DataType]]:
         """Reverse the transformation applied.
 
         Number of input tensors must align with the number of``data_keys``. If ``data_keys`` is not set, use
         ``self.data_keys`` by default.
         """
+        original_keys = None
+        if len(args) == 1 and isinstance(args[0], dict):
+            original_keys, data_keys, args = self._preproc_dict_data(args[0])
+
+        # args here should already be `DataType`
+        # NOTE: how to right type to: unpacked args <-> tuple of args to unpack
+        # issue with `self._preproc_dict_data` return args type
```

Comment:
how can i type this in the right way?

```
kornia/augmentation/container/augment.py:291: error: Argument 1 to "_validate_args_datakeys" of
"AugmentationSequential" has incompatible type
"*tuple[Tensor | list[Tensor] | Boxes | Keypoints | dict[str, Tensor | list[Tensor] | Boxes | Keypoints], ...]";
expected "Tensor | list[Tensor] | Boxes | Keypoints"  [arg-type]
            self._validate_args_datakeys(*args, data_keys=self.transform_op.data_keys)
```

Reply:
i do not know how to properly type this one, can I go with the type ignore here?